app/src/main/java/com/xmartlabs/scasas/geoquiz/ui/QuizActivity.java

```diff
@@ -46,15 +74,41 @@ private void setupQuizButton() {
     trueButon.setOnClickListener(new View.OnClickListener() {
       @Override
       public void onClick(View v) {
-        Toast.makeText(QuizActivity.this, R.string.incorrect_toast, Toast.LENGTH_SHORT).show();
+        Toast.makeText(QuizActivity.this, checkAnswer(trueButon.getText().toString()), Toast.LENGTH_SHORT).show();
       }
     });
     falseButton = (Button) findViewById(R.id.false_button);
     falseButton.setOnClickListener(new View.OnClickListener() {
       @Override
       public void onClick(View v) {
-        Toast.makeText(QuizActivity.this, R.string.correct_toast, Toast.LENGTH_SHORT).show();
+        Toast.makeText(QuizActivity.this, checkAnswer(falseButton.getText().toString()), Toast.LENGTH_SHORT).show();
       }
     });
+    nextButton = (Button) findViewById(R.id.next_button);
+    nextButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        if (currentIndex == questionBank.length - 1) {
```

Comment:
just for clarity purposes, consider in these cases extracting this into a new method.
```java
if (isLastQuestion()) {
```
is far more readable.
Just a thought